assets/airports/kbos.json

```diff
@@ -398,316 +528,8175 @@
-                "14":  ["_LOGAN14", "_LOGAN9-6"],
-                "15R": ["_LOGAN15R", "_LOGAN9-1"],
-                "22L": ["_LOGAN22L"],
-                "22R": ["_LOGAN22R"],
-                "27":  ["_LOGAN27", "_LOGAN9-2"],
-                "33L": ["_LOGAN33L", "_LOGAN9-3"]
+                "27": ["_BOS261002", "WYLYY", "CYOTI"]
             },
+            "body": [],
             "exitPoints": {
-                "MHT": ["MHT"],
-                "ENE": ["ENE"],
-                "PSM": ["PSM"],
-                "LFV": ["LFV"],
                 "ACK": ["ACK"],
-                "MVY": ["MVY"],
-                "PVD": ["PVD"],
-                "SEY": ["SEY"],
-                "CMK": ["CMK"],
                 "BAF": ["BAF"],
+                "BLZZR": ["BLZZR"],
+                "BOSOX": ["BOSOX"],
+                "BRUWN": ["BRUWN"],
+                "BURDY": ["BURDY"],
+                "CELTK": ["CELTK"],
                 "CTR": ["CTR"],
-                "CAM": ["CAM"],
                 "FRILL": ["FRILL"],
-                "FREDO": ["FREDO"],
-                "DUNKK": ["DUNKK"],
-                "BURDY": ["BURDY"],
-                "BOSOX": ["BOSOX"],
-                "NELIE": ["NELIE"],
                 "GLYDE": ["GLYDE"],
-                "HANAA": ["HANAA"]
+                "HYLND": ["HYLND"],
+                "LBSTA": ["LBSTA"],
+                "LFV": ["LFV"],
+                "LUCOS": ["LUCOS"],
+                "MHT": ["MHT"],
+                "NELIE": ["NELIE"],
+                "ONEPS": ["ONEPS"],
+                "PATSS": ["PATSS"],
+                "PSM": ["PSM"],
+                "REVSS": ["REVSS"],
+                "SEY": ["SEY"],
+                "SPENO": ["SPENO"],
+                "SSOXS": ["SSOXS"]
 
             },
             "draw": [
-                ["BOS", "MHT*"],
-                ["BOS", "ENE*"],
-                ["BOS", "PSM*"],
-                ["BOS", "LFV*"],
-                ["BOS", "ACK*"],
-                ["BOS", "MVY*"],
-                ["BOS", "PVD*"],
-                ["BOS", "SEY*"],
-                ["BOS", "CMK*"],
-                ["BOS", "BAF*"],
-                ["BOS", "CTR*"],
-                ["BOS", "CAM*"],
-                ["BOS", "FRILL*"],
-                ["BOS", "FREDO*"],
-                ["BOS", "DUNKK*"],
-                ["BOS", "BURDY*"],
-                ["BOS", "BOSOX*"],
-                ["BOS", "NELIE*"],
-                ["BOS", "GLYDE*"],
-                ["BOS", "HANAA*"]
+                ["_BOS261002", "WYLYY", "CYOTI"],
+                ["ACK*"], ["BAF*"], ["BLZZR*"],
+                ["BOSOX*"], ["BRUWN*"], ["BURDY*"],
+                ["CELTK*"], ["CTR*"], ["FRILL*"],
+                ["GLYDE*"], ["HYLND*"], ["LBSTA*"],
+                ["LFV*"], ["LUCOS*"], ["MHT*"],
+                ["NELIE*"], ["ONEPS*"], ["PATSS*"],
+                ["PSM*"], ["REVSS*"], ["SEY*"],
+                ["SPENO*"], ["SSOXS*"]
             ]
         }
     },
     "stars": {
```

Comment:
Will resume reviewing accuracy of STARs once the posted restrictions are corrected.